fishtest/tests/util.py

```diff
@@ -0,0 +1,7 @@
+from fishtest.rundb import RunDb
+
+def find_run():
```

Comment:
Maybe we could rename this `find_travis_run` or even have a function `find_user_run(userName)` which is called with `travis` as an argument.

Reply:
Good suggestion, will change that